packages/cli/lib/utils.js

```diff
@@ -105,7 +106,7 @@ exports.validateClassName = function(name) {
  * Validate project directory to not exist
  */
 exports.validateNotExisting = function(projDir) {
-  if (fs.existsSync(projDir)) {
+  if (fs.existsSync(untildify(projDir))) {
```

Comment:
Support for `~` is an entirely user-facing feature. I don't think it makes sense for an internal `util.validateNotExisting()` function to support it directly, especially when none of the other helpers support it.

Reply:
Looking at where this is actually used, I think the `untildify` call should be added to the validator in-line. Something like this:

```diff
--- a/packages/cli/lib/project-generator.js
+++ b/packages/cli/lib/project-generator.js
@@ -159,7 +159,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
           this.projectInfo.outdir == null ||
           // prompts if option was set to a directory that already exists
           utils.validateNotExisting(this.projectInfo.outdir) !== true,
-        validate: utils.validateNotExisting,
+        validate: input => utils.validateNotExisting(utils.untildify(input)),
         default: utils.toFileName(this.projectInfo.name),
       },
     ];
```

Reply:
+1 to what @rmg wrote. Can we revert this line and modify the `validate` function to call untildify before calling `validateNotExisting`?

Reply:
I had `validate: input => utils.validateNotExisting(utils.untildify(input))` before, but the coverage dropped a bit ( like what @rmg said, it mocks the prompts). I guess it's fine to decrease the coverage a bit?

Reply:
> I guess it's fine to decrease the coverage a bit?

Sure, it's ok in cases like this.